examples/jupyterhub-proxy.pid

```diff
@@ -0,0 +1 @@
+12533
```

Comment:
This is automatically generated, it doesn't make sense to add I think. It references a started process id.